operator/controllers/manifest_controller.go

```diff
@@ -35,13 +36,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/ratelimiter"
+	"sigs.k8s.io/controller-runtime/pkg/source"
 	"time"
 )
 
 const (
-	DefaultWorkersCount = 4
```

Comment:
Why the change? I'd rather increase this value to, say, 10 workers. Better yet, I'd expose it as a bootstrap flag with a default value of 10.

Reply:
Good idea, I will include a configurable flag to adjust it, defaulting to 1